tools/mmp/driver.cs

```diff
@@ -652,6 +652,11 @@ static Version MutateSDKVersionToPointRelease (Version rv)
 				if (rv.Minor == 11 && XcodeVersion >= new Version (7, 3))
 					return new Version (rv.Major, rv.Minor, 4);
 			}
+			// Since Version has wrong behavior:
+			// new Version (10, 14) < new Version (10, 14, 0) => true
+			// Force any unset revision to 0 instead of -1
+			if (rv.Revision == -1)
+				return new Version (rv.Major, rv.Minor, 0);
```

Comment:
would not this be solved if the previous condition included
```
if (rv.Minor == 14 && XcodeVersion >= new Version (10, 0))
	return new Version (rv.Major, rv.Minor, 0);
```
Wondering if we could be missing another case since the new check does not consider Xcode version

Reply:
I was trying to generally solve the issue so we didn't repeat in 10.15 without affecting existing behavior. 

Not sure on the last part. @rolfbjarne ?

Reply:
I think this is fine, it's a more general check than the previous ones, so it would work for future Xcode versions as well.